#! /bin/sh /usr/share/dpatch/dpatch-run
## 82_change_version_number.dpatch by  <tim@tim-VBox>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Upped the upstream version number

@DPATCH@

diff -r 9f8b70ab2429 configure.ac
--- a/configure.ac	Thu Apr 30 13:21:06 2009 +0700
+++ b/configure.ac	Thu Aug 20 15:45:23 2009 +0700
@@ -8,7 +8,7 @@
 
 SCIM_MAJOR_VERSION=1
 SCIM_MINOR_VERSION=4
-SCIM_MICRO_VERSION=7
+SCIM_MICRO_VERSION=8
 
 SCIM_VERSION=$SCIM_MAJOR_VERSION.$SCIM_MINOR_VERSION.$SCIM_MICRO_VERSION
 
